util/llpcUtil.cpp

```diff
@@ -154,4 +154,37 @@ bool CreateDirectory(
 #endif
 }
 
+// =====================================================================================================================
+// Helper macro
+#define CASE_CLASSENUM_TO_STRING(TYPE, ENUM) \
```

Comment:
Put this macro and the function GetResourceMappingNodeTypeName in llpcInternal.cpp
For llpcUtil.cpp, we often place simple graphics-independent functions, such as math functions.